renderdoc/driver/shaders/dxbc/dxbc_disassemble.h

```diff
@@ -687,6 +687,8 @@ class DXBCFile;
 struct ASMIndex;
 struct ASMDecl;
 
+char *toString(OpcodeType op);
```

Comment:
I'd prefer just to print the opcode as a number rather than making this public. The string isn't necessary for any error messages since it's not going to the user, it's indicating a coding error so anyone who sees it can look up the opcode by value.

Reply:
Good point, reverted this bit.